### DISCUSSION TITLE
How to implement

### DISCUSSION THREAD

**Discussion by MikeEitel** [frangoteam/FUXA#1275] [Q&A]
Hi
It is my first try with FUXA. Beside of that I have many years experience with lots of SCADA's.
I have quite a lot of devices in my home automation and use node red to send structured data that i want to use in fuxa via a dedicated mosquitto server. 

I have structured them in mqtt like this:
048/temp0
048/hum
048/watchdog
049/temp0
049/hum
049/watchdog

041/0/on
041/1/on
041/0/temp0
041/0/temp1
041/1/temp0
041/1/temp1

I want to show them on a page in a "templet" way, means I want to have a "template* for every type of device.
I'm used to you give a part of the address  ( fx 048 ) to a "template call"  that fulfills the the rest to combine it to a complete variable name...

I looked into the wiki and it seems to be possible but I do not get it. 

P.S.
I understand how to connect a mqtt to a fuxa-variable-

**Comment by unocelli** [comment]
Hi, have you try with export/import function, then edit it as CSV file
